diff -Naur kdebase-trinity-3.5.13.2.orig/ConfigureChecks.cmake kdebase-trinity-3.5.13.2/ConfigureChecks.cmake
--- kdebase-trinity-3.5.13.2.orig/ConfigureChecks.cmake	2012-12-14 02:45:03.000000000 +0000
+++ kdebase-trinity-3.5.13.2/ConfigureChecks.cmake	2019-12-17 01:10:21.537390099 +0000
@@ -234,31 +234,31 @@
 
 
 # dbus (kdm, ksmserver)
-if( BUILD_KDM OR BUILD_KSMSERVER )
-
-  pkg_search_module( DBUS dbus-1 )
-  if( NOT DBUS_FOUND )
-    tde_message_fatal( "dbus-1 is required, but was not found on your system" )
-  endif( )
-
-endif( )
-
-
-# dbus-tqt (kdm, ksmserver, kioslaves(media))
-if( BUILD_KDM OR BUILD_KSMSERVER OR (BUILD_KIOSLAVES AND WITH_HAL) )
-  # check for dbus-tqt
-  pkg_check_modules( DBUS_TQT REQUIRED dbus-tqt )
-  tde_save( CMAKE_REQUIRED_INCLUDES CMAKE_REQUIRED_LIBRARIES )
-  set( CMAKE_REQUIRED_INCLUDES ${DBUS_TQT_INCLUDE_DIRS} ${TQT_INCLUDE_DIRS} ${QT_INCLUDE_DIRS})
-  set( CMAKE_REQUIRED_LIBRARIES ${DBUS_TQT_LDFLAGS} ${TQT_LDFLAGS} ${QT_LDFLAGS} )
-  check_cxx_source_compiles("
-    #include <tqt.h>
-    #include <dbus/connection.h>
-    int main(int, char**) { return 0; } "
-    HAVE_DBUS_QT3_07 )
-  tde_restore( CMAKE_REQUIRED_INCLUDES CMAKE_REQUIRED_LIBRARIES )
-  if( NOT HAVE_DBUS_QT3_07 )
-    tde_message_fatal( "dbus-tqt is required, but was not found on your system" )
-  endif( )
-
-endif( )
+#if( BUILD_KDM OR BUILD_KSMSERVER )
+#
+#  pkg_search_module( DBUS dbus-1 )
+#  if( NOT DBUS_FOUND )
+#    tde_message_fatal( "dbus-1 is required, but was not found on your system" )
+#  endif( )
+#
+#endif( )
+#
+#
+## dbus-tqt (kdm, ksmserver, kioslaves(media))
+#if( BUILD_KDM OR BUILD_KSMSERVER OR (BUILD_KIOSLAVES AND WITH_HAL) )
+#  # check for dbus-tqt
+#  pkg_check_modules( DBUS_TQT REQUIRED dbus-tqt )
+#  tde_save( CMAKE_REQUIRED_INCLUDES CMAKE_REQUIRED_LIBRARIES )
+#  set( CMAKE_REQUIRED_INCLUDES ${DBUS_TQT_INCLUDE_DIRS} ${TQT_INCLUDE_DIRS} ${QT_INCLUDE_DIRS})
+#  set( CMAKE_REQUIRED_LIBRARIES ${DBUS_TQT_LDFLAGS} ${TQT_LDFLAGS} ${QT_LDFLAGS} )
+#  check_cxx_source_compiles("
+#    #include <tqt.h>
+#    #include <dbus/connection.h>
+#    int main(int, char**) { return 0; } "
+#    HAVE_DBUS_QT3_07 )
+#  tde_restore( CMAKE_REQUIRED_INCLUDES CMAKE_REQUIRED_LIBRARIES )
+#  if( NOT HAVE_DBUS_QT3_07 )
+#    tde_message_fatal( "dbus-tqt is required, but was not found on your system" )
+#  endif( )
+#
+#endif( )
diff -Naur kdebase-trinity-3.5.13.2.orig/ksmserver/global.h kdebase-trinity-3.5.13.2/ksmserver/global.h
--- kdebase-trinity-3.5.13.2.orig/ksmserver/global.h	2012-05-24 01:35:16.000000000 +0000
+++ kdebase-trinity-3.5.13.2/ksmserver/global.h	2019-12-17 01:14:22.375256493 +0000
@@ -10,8 +10,8 @@
 #define KSMVendorString "KDE"
 #define KSMReleaseString "1.0"
 
-#ifdef USE_QT4
-#define NO_QT3_DBUS_SUPPORT
-#endif
+//#ifdef USE_QT4
+//#define NO_QT3_DBUS_SUPPORT
+//#endif
 
 #endif
diff -Naur kdebase-trinity-3.5.13.2.orig/ksmserver/shutdowndlg.h kdebase-trinity-3.5.13.2/ksmserver/shutdowndlg.h
--- kdebase-trinity-3.5.13.2.orig/ksmserver/shutdowndlg.h	2012-09-16 12:52:04.000000000 +0000
+++ kdebase-trinity-3.5.13.2/ksmserver/shutdowndlg.h	2019-12-17 01:14:06.463390594 +0000
@@ -32,11 +32,11 @@
 
 #include <config.h>
 
-#ifndef NO_QT3_DBUS_SUPPORT
-/* We acknowledge the the dbus API is unstable */
-#define DBUS_API_SUBJECT_TO_CHANGE
-#include <dbus/connection.h>
-#endif // NO_QT3_DBUS_SUPPORT
+//#ifndef NO_QT3_DBUS_SUPPORT
+///* We acknowledge the the dbus API is unstable */
+//#define DBUS_API_SUBJECT_TO_CHANGE
+//#include <dbus/connection.h>
+//#endif // NO_QT3_DBUS_SUPPORT
 
 #ifdef COMPILE_HALBACKEND
 #include <hal/libhal.h>
diff -Naur kdebase-trinity-3.5.13.2.orig/ksmserver/startupdlg.h kdebase-trinity-3.5.13.2/ksmserver/startupdlg.h
--- kdebase-trinity-3.5.13.2.orig/ksmserver/startupdlg.h	2012-09-16 12:52:04.000000000 +0000
+++ kdebase-trinity-3.5.13.2/ksmserver/startupdlg.h	2019-12-17 01:13:46.399558250 +0000
@@ -32,11 +32,11 @@
 
 #include <config.h>
 
-#ifndef NO_QT3_DBUS_SUPPORT
-/* We acknowledge the the dbus API is unstable */
-#define DBUS_API_SUBJECT_TO_CHANGE
-#include <dbus/connection.h>
-#endif // NO_QT3_DBUS_SUPPORT
+//#ifndef NO_QT3_DBUS_SUPPORT
+///* We acknowledge the the dbus API is unstable */
+//#define DBUS_API_SUBJECT_TO_CHANGE
+//#include <dbus/connection.h>
+//#endif // NO_QT3_DBUS_SUPPORT
 
 #ifdef COMPILE_HALBACKEND
 #include <hal/libhal.h>
