diff -Naur kdebase-trinity-3.5.13.2.orig/kdesktop/lock/lockprocess.cc kdebase-trinity-3.5.13.2/kdesktop/lock/lockprocess.cc
--- kdebase-trinity-3.5.13.2.orig/kdesktop/lock/lockprocess.cc	2013-05-14 19:43:53.000000000 +0000
+++ kdebase-trinity-3.5.13.2/kdesktop/lock/lockprocess.cc	2019-12-16 23:04:18.654478808 +0000
@@ -22,7 +22,7 @@
 
 #include <config.h>
 
-#include <execinfo.h>
+//#include <execinfo.h>
 
 #include "lockprocess.h"
 #include "lockdlg.h"
@@ -143,7 +143,7 @@
 
 void print_trace()
 {
-#ifdef WITH_KDESKTOP_LOCK_BACKTRACE
+/*#ifdef WITH_KDESKTOP_LOCK_BACKTRACE
 	void *array[10];
 	size_t size;
 	char **strings;
@@ -159,7 +159,7 @@
 	}
 
 	free (strings);
-#endif
+#endif*/
 }
 
 static void segv_handler(int)
